packages/stargate/src/queries/staking.ts

```diff
@@ -165,7 +167,7 @@ export function setupStakingExtension(base: QueryClient): StakingExtension {
           });
           return toObject(response);
         },
-        validators: async (status: string, paginationKey?: Uint8Array) => {
+        validators: async (status: keyof typeof BondStatus, paginationKey?: Uint8Array) => {
```

Comment:
Nice type magic. So I guess we could also do `type BondStatusString = Exclude<keyof typeof BondStatus, "BOND_STATUS_UNSPECIFIED">`.

I think it would be nice to accept the same argument here and above. "BOND_STATUS_UNSPECIFIED" should never be used.

Reply:
Excluding `BOND_STATUS_UNSPECIFIED` makes sense. I actually thought I'd removed this from here because I couldn't use it in the other place for some obscure type reason I didn't investigate too thoroughly.